packages/openapi-typescript/test/transform/schema-object/object.test.ts

```diff
@@ -46,7 +54,8 @@ describe("transformSchemaObject > object", () => {
         },
         want: `{
     property?: boolean;
-    [key: string]: string | undefined;
+} & {
+    [key: string]: string;
```

Comment:
Note: this intersection seems a little awkward, but is necessary for TypeScript to work properly in most cases (here, you’d get some error like “property `string` not assignable to type `boolean | undefined`”).